apps/meteor/client/views/admin/oauthApps/OAuthAppsPage.tsx

```diff
@@ -19,7 +19,7 @@ const OAuthAppsPage = (): ReactElement => {
 	return (
 		<Page flexDirection='row'>
 			<Page>
-				<Page.Header title={t('OAuth_Applications')}>
```

Comment:
since this token is not being used, remove it from i18n files, check all replaced tokens, pelase

Reply:
It was already removed from the english dictionary. By default it will be excluded from other folders.